MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/ColorBarWidget.h

```diff
@@ -56,11 +60,15 @@ class EXPORT_OPT_MANTIDQT_MANTIDWIDGETS ColorBarWidget : public QWidget {
   void setMaximum(double max);
   void setRenderMode(bool rendering);
 
+  /// Set which checkboxes are shown in the window
+  void setCheckBoxMode(CheckboxStrategy strategy);
```

Comment:
Nice. don't hesitate to add bits of documentation like this when you see they're missing.